process_pr.py

```diff
@@ -55,6 +55,7 @@ def format(s, **kwds): return s % kwds
 EXTRA_TESTS = "gpu|threading|profiling|none"
 MULTILINE_COMMENTS_MAP = {
               "workflow(s|)(_gpu|_threading|)":  [format('^%(workflow)s(\s*,\s*%(workflow)s|)*$', workflow= WF_PATTERN),       "MATRIX_EXTRAS"],
+              "workflow(s|)(_profiling|)":  [format('^%(workflow)s(\s*,\s*%(workflow)s|)*$', workflow= WF_PATTERN),       "PROFILING_WORKFLOWS"],
```

Comment:
change it to `workflow(s|)_profiling` otherwise `workflows = num` will set PROFILING_WORKFLOWS instead of MATRIX_EXTRAS